application/src/test/java/org/thingsboard/server/controller/BaseEntityRelationControllerTest.java

```diff
@@ -432,7 +432,7 @@ public void testFindRelationsByFromQuery() throws Exception {
                 1, true
         ));
         query.setFilters(Collections.singletonList(
-                new RelationEntityTypeFilter("CONTAINS", List.of(EntityType.DEVICE))
+                new RelationEntityTypeFilter("CONTAINS", List.of(EntityType.DEVICE), false)
```

Comment:
Please create one more constructor that will have 'false' as the default value. This will make all these redundant changes to go away.